### ISSUE TITLE
[BUG] Packages go missing after restarting WingetUI 

### ISSUE THREAD

**Issue by skanda890** [marticliment/WingetUI#2007]
### Please confirm these before moving forward

- [X] I have searched for my issue and have not found a work-in-progress/duplicate/resolved issue.
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
- [X] I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section for solutions
- [X] This issue is about a bug (if it is not, please use the [correct template](https://github.com/marticliment/WingetUI/issues/new/choose))

### Describe your issue

Packages go missing after restarting WingetUI.

### Steps to reproduce the issue

Restart WingetUI, and check Installed Packages.

### WingetUI Log

https://1drv.ms/t/s!Aqq-Je19xlrfiCCnr0Kd5AyNP9lu?e=gbEyh7

### Package Managers Logs

https://1drv.ms/t/s!Aqq-Je19xlrfiCGg_2rP9XkcB2ZY?e=6N01Pv

**Comment by skanda890** [comment]
This was after #2006

**Comment by skanda890** [comment]
I had to refresh Installed Packages, but why does this happen? Make it so that it will always refresh the Installed Packages after restarting WingetUI.

**Comment by mbunkus** [comment]
Same issue for me. Refreshing the list of installed packages doesn't help. The list of installed applications is very, very short.

Today I installed Tabby & Alacritty with WingetUI. Gave both of them a good try. Decided to stay with Alacritty. Had to reboot for unrelated reasons. After the reboot neither application appears in the list of installed applications, even though they're both still installed.

WingetUI 3.0.1, Windows 11.

[Log file](https://www.bunkus.org/misc/wingetui-issue-2007/wingetui-log-202403222202.txt)

![2024-03-22 21_57_32- BUG  Packages go missing after restarting WingetUI · Issue #2007 · marticliment](https://github.com/marticliment/WingetUI/assets/414282/08a7fbed-311c-4cb5-aacd-143e73d76f87)

**Comment by marticliment** [comment]
Hi @mbunkus, your issue is different. As I can see by your logs, some WinGet packages are failing to be parsed. 

I would appreciate if you could open a new issue for this

**Comment by mbunkus** [comment]
Thanks for the quick feedback. I'll open a new one right away.

**Comment by skanda890** [comment]
This time I don't think WingetUI restarted, so today I didn't have to refresh WingetUI.

**Comment by marticliment** [comment]
Version of WingetUI?

**Comment by skanda890** [comment]
> Version of WingetUI?

3.02

**Comment by marticliment** [comment]
Please try 3.1.0 Alpha 1
